File: httpsnippet/src/test/resources/output/ruby/nethttp/multipart-form-data.rb
require 'uri'
require 'net/http'

url = URI("http://mockbin.com/har")

http = Net::HTTP.new(url.host, url.port)

request = Net::HTTP::Post.new(url)
request["content-type"] = "multipart/form-data"

response = http.request(request)
puts response.read_body

File: httpsnippet/src/test/resources/output/node/request/multipart-form-data.js
var request = require("request");

var options = {"headers":{"content-type":"multipart/form-data"},"method":"POST","formData":{"foo":"bar"},"url":"http://mockbin.com/har"};

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});
